src/routes/user.routes.ts

```diff
@@ -19,6 +19,6 @@ userRouter
 	.route("/users/:id")
 	.patch(validateRequest(patchUserSchema), deserializeUser, requireSameUser, patchUserHandler);
 
-userRouter.get("/users/marketing-emails/unsubscribe/:id", updateUnsubcribeUser);
+userRouter.get("/users/marketing-emails/unsubscribe/:id", validateRequest(updateUnsubscribeUserSchema),deserializeUser, updateUnsubscribeUser)
```

Comment:
- There are no inputs to this route, so you do not need a schema.

- Use `.route().get()` instead